### PR TITLE
try 3 at unscheduled in 76x

### PR DESCRIPTION
**Pull request by davidlange6**
Combining two PRs
#10054 (cherry picked to 76x) + #10457

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by davidlange6** [comment]
@cmsbuild, please test

**Comment by davidlange6** [comment]
unit test problem is unrelated

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -1026,13 +886,18 @@ def genvalid(fragment,d,suffix='all',fi='',dataSet=''):
...
-                 '--datatier':'GEN-SIM-RECO,DQMIO',
-                 '--eventcontent':'RECOSIM,DQM',
-                 '--customise' : 'SLHCUpgradeSimulations/Configuration/postLS1Customs.customisePostLS1'
-                 }
+step3Up2015Defaults = {
+    #'-s':'RAW2DIGI,L1Reco,RECO,EI,VALIDATION,DQM',
+    '-s':'RAW2DIGI,L1Reco,RECO,EI,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@miniAODDQM',
+    '--runUnscheduled':'',
+    '--conditions':'auto:run2_mc', 
+    '--magField'    : '38T_PostLS1',
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: why are we overriding mag field from xml here? (and in all other places)

**Comment by slava77** [comment]
I ran our reco tests on this
- the set of RECO products (in the RECO output file) is the same before and after
- fwlite-based comparisons in several MC (run1 and run2) workflows are showing small differences in the muon fits
    - standAloneSETMuons_UpdatedAtVtx and muonsFromCosmics are the most noticeably affected (appear in both single-mu and in "jetty" workflows as TTbarPU wf25202p0) ... looking at the config, I think it's from the "38T_PostLS1" in the updated matrix cmsDriver commands [the baseline uses DB]
- DQM plots have some oddities:
    - JetMET/Run summary cleanup and vertices histograms have twice the number of entries with this PR

The magnetic field issue should be fixed

**Comment by boudoul** [comment]
@slava77 , thanks for the report, I will fix the MagFied by today. 
I'll try to understand what could be the origin of the JetMET change

**Pull request merged**